src/Phase.jl

```diff
@@ -41,6 +43,24 @@ export IdealDiluteSolution
 end
 export HomogeneousCatalyst
 
+"""
+construct the stochiometric matrix for the reactions and the reaction molecule # change
+"""
+function getstoichmatrix(spcs,rxns)
+    M = zeros(length(rxns),length(spcs))
```

Comment:
For a model generated by RMG with ~50000 reactions and ~1000 reactants, I guess the RAM usage can be around 400 MB. Although it is not quite big since we are using sparse matrix anyway, can we not use zeros(length(rxns), length(spcs)) but simply create three lists and feed them to `sparse()`?

Reply:
fixed, used spzeros instead of zeros